tutorials/06_Advanced_ChangeInputs.md

```diff
@@ -67,11 +67,16 @@ If you are developing in R using VS Code, you can connect to the cluster via ssh
 - [Using R in the cluster with VSCode](https://gitlab.pik-potsdam.de/pascalfu/bettercode-vscode/-/blob/main/R_VScode_in_the_cluster.md)
 - [How to connect VScode to the cluster using ssh](https://github.com/pik-piam/discussions/discussions/4)
 
-### Build an up-to-date cache on the cluster for local development
+**Important: Keep in mind that heavy workloads must not be executed on login nodes and should be passed to compute nodes instead.** Make sure to read the section ["A word of caution on using the cluster"](https://gitlab.pik-potsdam.de/pascalfu/bettercode-vscode/-/blob/main/R_VScode_in_the_cluster.md#a-word-of-caution-on-using-the-cluster) before working in the cluster.
 
-If you want to develop a madrat package locally, you might want to work with an up-to-date cache to avoid gathering all source data locally and re-running lengthy calculations on your own computer.
+### Get an up-to-date cache on the cluster for local development
 
-The default cache folder on the cluster is too large for this purpose, but you can create your own cache on the cluster and download it. To do so, follow the steps 2) and 3) under [How to update input data](#how-to-update-input-data), but make further adjustments to the `start.R` script:
+If you want to develop a madrat package locally, you might want to work with an up-to-date cache to avoid gathering all source data locally and re-running lengthy calculations on your own computer. The default cache folder on the cluster is too large for this purpose, but there are two ways to creat your own cache and download it. 
+
+**Option 1**: You can use the tool `idrcp` to achieve this. The tool extracts all cache files either written to or read from listed in the logfile of a REMIND input data revision and stores them in an archive you can download. You just need to pass the path to a input data revision, e.g. `idrcp /p/projects/rd3mod/inputdata/output/rev6.99_remind.tgz`. (The script looks up the cache folder used for this revision in the log file of the archive and then extracts all cache files mentioned in the log from that folder and stores them in an archive.)
```

Comment:
`rev6.99_remind.tgz` is a file name that will not be generated by `madrat`, so people will stumble over that.

Suggestion:
> You need to pass the path to a input data revision, e.g. `idrcp /p/projects/rd3mod/inputdata/output/rev6.51_62eff8f7_remind.tgz`.  Be sure to select a `62eff8f7` file, as that region mapping (`H12`) is generated first, and subsequent region mappings (`2b1450bc`/`EU21`) only re-aggregate the results without using the cache.  Also, select a `remind` or a `validation` file, depending on which cache you need – basically whether the function you are working on is called from `mrremind::fullREMIND()` or `mrremind::fullVALIDATIONREMIND()`.

Reply:
Thanks, very useful. Will be added.

Reply:
Yeah, but also untrue, as we discovered.

Reply:
Because there are `2b1450bc` archives with proper cache directories
```

/p/tmp/pehl/Remind/output (develop| M)[1]$ tar -Oxf /p/projects/rd3mod/inputdata/output/rev6.51personal-cache_2b1450bc_remind.tgz ./diagnostics.log | grep -m 1 "cachefolder\s*->"
   cachefolder         -> "/p/tmp/benke/IEA_651"
```
When there is a cache directory, and when there is just a puc is an unsolved mystery.

Reply:
Ah, I see. So to rephrase with my own words: we cannot tell from the name of the archive if it uses a cache directory or a puc. 

So I guess I should write that it only works when the archive uses a proper cache directory, not a puc directory=